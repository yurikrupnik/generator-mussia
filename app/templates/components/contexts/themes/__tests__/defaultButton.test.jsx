import React from 'react';
import DefaultButton from '../defaultButton';

const {
    it,
    expect,
    shallow
} = global;

it('renders <DefaultButton /> component', () => {
    const wrapper = shallow(
        <DefaultButton
            theme={{ foreground: '#000000', background: '#eeeeee', color: 'red' }}
            toggleTheme={() => {}}
        />
    );
    expect(wrapper).toMatchSnapshot();
});
