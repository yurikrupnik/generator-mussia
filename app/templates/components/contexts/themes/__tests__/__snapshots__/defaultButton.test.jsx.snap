// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <DefaultButton /> component 1`] = `
<div>
  <button
    onClick={[Function]}
    style={
      Object {
        "background": "#eeeeee",
      }
    }
    type="button"
  >
    Toggle
  </button>
</div>
`;
